mysql-test/suite/innodb/t/buf_pool_read_request.test

```diff
@@ -0,0 +1,16 @@
+--source include/have_innodb.inc
+--source include/have_sequence.inc
+--echo #
+--echo # MDEV-32390 innodb_buffer_pool_read_requests always 0
+--echo #
+create table t1(f1 int not null, f2 int not null)engine=innodb;
+insert into t1 select seq, seq from seq_1_to_16384;
```

Comment:
Instead of creating a possibly unreliable test for this, could we simply add a single SQL statement to an existing test?
```sql
SELECT CAST(variable_value AS INTEGER)>0 FROM INFORMATION_SCHEMA.GLOBAL_STATUS WHERE VARIABLE_NAME='innodb_buffer_pool_read_requests';
```
Or add two statements to an existing test that is restarting the server:
```diff
diff --git a/mysql-test/suite/innodb/t/innodb_force_recovery.test b/mysql-test/suite/innodb/t/innodb_force_recovery.test
index 2368af76f09..30364c06bb3 100644
--- a/mysql-test/suite/innodb/t/innodb_force_recovery.test
+++ b/mysql-test/suite/innodb/t/innodb_force_recovery.test
@@ -21,8 +21,18 @@ SET GLOBAL innodb_fast_shutdown = 0;
 --source include/restart_mysqld.inc
 let $status=`SHOW ENGINE INNODB STATUS`;
 
+SELECT CAST(variable_value AS INTEGER) INTO @read1
+FROM INFORMATION_SCHEMA.GLOBAL_STATUS
+WHERE VARIABLE_NAME='innodb_buffer_pool_read_requests';
+
 select * from t1;
 
+SELECT CAST(variable_value AS INTEGER) INTO @read2
+FROM INFORMATION_SCHEMA.GLOBAL_STATUS
+WHERE VARIABLE_NAME='innodb_buffer_pool_read_requests';
+
+SELECT @read1>0, @read2>@read1;
+
 begin;
 insert into t1 values(2, 3);
 rollback;
```